### ISSUE TITLE
Command-line updater

### ISSUE THREAD

**Issue by dorianim** [meeting-room-booking-system/mrbs-code#20]
**Is your feature request related to a problem? Please describe.**
I'm currently working on a docker container for mrbs and I would like the container to be able to auto-update.

**Describe the solution you'd like**
It would be nice to have some script which can be called to upgrade mrb if necessary.

**Comment by jberanek** [comment]
Should be fairly easy to rework upgrade.inc to turn the main code into a function, which could be fun from a CLI wrapper

**Comment by campbell-m** [comment]
Does [this](https://github.com/meeting-room-booking-system/mrbs-code/pull/2) help at all?

**Comment by jberanek** [comment]
The CLI wrapper could then support reading the admin username and password from environment variables.

**Comment by dorianim** [comment]
> The CLI wrapper could then support reading the admin username and password from environment variables.

Yep, that would be great!

By the way, you can see my work here:
https://github.com/dorianim/mrbs-docker
(It's not finished yet, though)

**Comment by being-peace** [comment]
@dorianim By the way, are you aware of the minimalistic docker approach, here: https://github.com/meeting-room-booking-system/mrbs-code/tree/main/docker_app

**Comment by dorianim** [comment]
@being-peace Yes, but it is not what I did. I have enclosed the mrbs source code into the image to automate the upgrade process. Like this, a user just has to pull the updated image and is ready to go.
I am personally using watchtower to automate that process completely. This is also the reason for this feature request, because it will completely remove the need for any user interaction on upgrades.
